src/main/java/org/broadinstitute/hellbender/utils/gcs/BucketUtils.java

```diff
@@ -390,7 +390,7 @@ public static String getPathWithoutBucket(String path) {
      * To transform other types of string URLs into Paths, use IOUtils.getPath instead.
      */
     public static java.nio.file.Path getPathOnGcs(String gcsUrl) {
-        final String[] split = gcsUrl.split("/");
+        final String[] split = gcsUrl.split("/", 4);
```

Comment:
Sorry, I just hadn't pushed that change from 4 to -1 up yet. I'll push it with the rest of the PR-comment-related-fixes.